src/containers/my-courses/my-courses-container/MyCorsesCardsList.styles.ts

```diff
@@ -2,11 +2,14 @@ export const styles = {
   wrapper: {
     display: 'grid',
     gridTemplateColumns: {
-      xs: '1fr',
-      md: 'repeat(2, 1fr)',
-      lg: 'repeat(3, 1fr)'
+      xs: 'minmax(0, 1fr)',
+      md: 'repeat(2, minmax(0, 1fr))',
+      lg: 'repeat(3, minmax(0, 1fr))'
     },
     gap: '24px',
     mb: '24px'
+  },
+  card: {
```

Comment:
Try to set `height` for `card` styles as I write above and not to create a new property